### PR TITLE
Submit non-US-specific strings for localisation

### PR DESCRIPTION
**Pull request by Vinnl**
# References: 
Jira: MNTOR-3676

I went through the strings in `dashboard.ftl` and `fix.ftl` to isolate the ones that were not specific to the US. Note that some strings were visible to everyone but referenced things only available in the US - thus, this PR also includes some minor code and string changes to accommodate that.

### CODE REVIEW

**Review by flodolo — CHANGES_REQUESTED** [review]
Leaving a few initial notes.

As commented on Slack, I don't expect content to be localized at this point, and this exposed hundreds of strings for localization.

File: locales/en/dashboard.ftl
Comment (flodolo) [reply locales/en/dashboard.ftl:98]: The irony: https://github.com/mozilla/blurts-server/pull/3493#discussion_r1342663157

File: locales-pending/dashboard-premium.ftl
Comment (flodolo) [inline locales-pending/dashboard-premium.ftl:1]: Note: we should add a license header to all the FTL files.

```
# This Source Code Form is subject to the terms of the Mozilla Public
# License, v. 2.0. If a copy of the MPL was not distributed with this
# file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

File: locales-pending/fix-premium.ftl
Comment (flodolo) [inline locales-pending/fix-premium.ftl:92]:
```suggestion
data-broker-profiles-estimated-time = { $estimated_time -> 
   *[other] Est. time to complete: { $estimated_time } mins
}
```

Still needs a plural, given "mins".

File: locales-pending/fix-premium.ftl
Comment (flodolo) [inline locales-pending/fix-premium.ftl:102]:
```suggestion
# the label inside <label_line_2> will be shown in smaller font. Second line of the label
```

File: locales/en/app.ftl
Comment (flodolo) [inline locales/en/app.ftl:914]: nit: obsolete is probably more accurate than deprecated.

File: locales/en/dashboard.ftl
Comment (flodolo) [inline locales/en/dashboard.ftl:52]: This is another string with multiple plurals.

File: locales/en/dashboard.ftl
Comment (flodolo) [inline locales/en/dashboard.ftl:59]: Is this an action (e.g. button) or a title?

File: locales/en/fix.ftl
Comment (flodolo) [inline locales/en/fix.ftl:61]:
```suggestion
high-risk-breach-estimated-time = { $estimated_time -> 
   *[other] Your estimated time: { $estimated_time }+ minutes
}
```

File: locales/en/fix.ftl
Comment (flodolo) [inline locales/en/fix.ftl:74]: ASAP is colloquial: can we spell it out, or add a comment to explain it?

File: locales/en/fix.ftl
Comment (flodolo) [inline locales/en/fix.ftl:99]: Same comment here for ASAP

File: locales/en/fix.ftl
Comment (flodolo) [inline locales/en/fix.ftl:134]: Phone masks are not available out of US, is this hidden?

File: locales/en/fix.ftl
Comment (flodolo) [inline locales/en/fix.ftl:183]:
```suggestion
leaked-passwords-estimated-time = { $estimated_time -> 
   *[other] Est. time to complete: { $estimated_time } mins per site
}
```

File: locales-pending/dashboard-premium.ftl
Comment (pdehaan) [inline locales-pending/dashboard-premium.ftl:2]: Q: Per @rhelmer's recent #3934 PR, should these be `-premium` or `-plus`?

Comment (rhelmer) [reply locales-pending/dashboard-premium.ftl:2]: I believe that Flod suggested `brand-monitor-plus`, we were holding back landing that PR until we were closer to shipping but I believe we decided to just land it which should make this easier.

Comment (flodolo) [reply locales-pending/dashboard-premium.ftl:2]: @rhelmer 
Not sure I understand. 

> we decided to just land it which should make this easier.

"it" is which PR?

**Review by flozia — APPROVED** [review]
Thanks! The code changes are looking good to me.

File: locales/en/bento.ftl
Comment (Vinnl) [reply locales/en/bento.ftl]: Ah, so you mean no brand strings for Fx desktop and Fx mobile? feef94913

File: locales/en/dashboard.ftl
Comment (Vinnl) [reply locales/en/dashboard.ftl]: Heh, guess I should've insisted more :sweat_smile: 

Split up in 4d5e6654e.

File: locales/en/fix.ftl
Comment (Vinnl) [reply locales/en/fix.ftl:92]: It looks like credit freeze is the advice we give if your SSN has been breached - in the US it's apparently possible to open loans or credit cards with just an SSN.

We might be able to hide this page outside the US, and thus remove these strings from this file - I've asked about that [here](https://mozilla.slack.com/archives/C0460QALW69/p1704280574265349?thread_ts=1704236102.962079&cid=C0460QALW69).

File: locales-pending/dashboard-premium.ftl
Comment (Vinnl) [reply locales-pending/dashboard-premium.ftl:1]: Good call: 7e8339939 and dbf502ff9.

File: locales-pending/fix-premium.ftl
Comment (Vinnl) [reply locales-pending/fix-premium.ftl]: af7196339

File: locales-pending/fix-premium.ftl
Comment (Vinnl) [reply locales-pending/fix-premium.ftl]: d75018fcc

File: locales/en/app.ftl
Comment (Vinnl) [reply locales/en/app.ftl]: Ah, I've been using `# Deprecated` for all strings I'm intending to remove so far; should I switch to using Obsolete in the future, or is that just for a specific situation in which we're planning to remove strings?

Changed all instances for now: 53ef0aee9

File: locales/en/dashboard.ftl
Comment (Vinnl) [reply locales/en/dashboard.ftl]: Since this needs to be split in three, I asked about how to reformulate this [here](https://mozilla.slack.com/archives/C0460QALW69/p1704282394779109?thread_ts=1704214729.664269&cid=C0460QALW69).

It also looks like it will need to be split into a US and non-US version, since it mentions data brokers.

File: locales/en/dashboard.ftl
Comment (Vinnl) [reply locales/en/dashboard.ftl:59]: eb582afb5

File: locales/en/fix.ftl
Comment (Vinnl) [reply locales/en/fix.ftl]: ceb70ceba

File: locales/en/fix.ftl
Comment (Vinnl) [reply locales/en/fix.ftl]: Looking up [the previous discussion](https://mozilla.slack.com/archives/C0460QALW69/p1695398920252159?thread_ts=1695329076.892809&cid=C0460QALW69) on this, it looks like the preference was to use ASAP on (currently) US-only pages, and spell it out elsewhere. So since this is visible outside the US, I'm assuming that changing it is fine: 0f695d634

File: locales/en/fix.ftl
Comment (Vinnl) [reply locales/en/fix.ftl]: 0f695d634

File: locales/en/fix.ftl
Comment (Vinnl) [reply locales/en/fix.ftl]: [It's not clear if it should be](https://docs.google.com/document/d/1-_KSIEW07pbT3eO-bS4JcnrTstCduucc1omdo2RXzuM/edit?disco=AAABDlbwnZw), but I've hidden it for now: 3b7cd9dee.

It might be reverted though, since there is the ability to sign up for the waiting list.

File: locales/en/fix.ftl
Comment (Vinnl) [reply locales/en/fix.ftl]: e95509d05

File: src/app/components/client/Chart.tsx
Comment (Vinnl) [reply src/app/components/client/Chart.tsx]: And actually that env var aleady existed without `NEXT_PUBLIC`. I've just added it for now, since I do expect that change to get merged in in the short term: 5a7766d60.

**Comment by Vinnl** [comment]
Changes I made not in reply to comments here:

6838dc296

"Here's what _you_ fixed" for non-Premium, i.e. when we didn't do anything.

image

f49a481ae

Add a third paragraph to the dialog on the "Action needed" tab.

image

File: locales/en/dashboard.ftl
Comment (flodolo) [inline locales/en/dashboard.ftl:130]: There is no variable in the string.

File: locales-pending/dashboard-premium.ftl
Comment (flodolo) [reply locales-pending/dashboard-premium.ftl:1]: We should add it also to the ones in `locales-pending` though.

File: src/app/components/client/Chart.tsx
Comment (Vinnl) [reply src/app/components/client/Chart.tsx]: Oh heh, of course the tests then fail. But I can just make the same change here and Git can reconcile it whenever the second PR gets merged: d6b39de0e

File: locales-pending/dashboard-premium.ftl
Comment (Vinnl) [reply locales-pending/dashboard-premium.ftl:5]: Added in 1aaa95cd8 for all existing FTL files that weren't touched in the PR as well. And I figured I'd just commit the removal of trailing spaces that got applied when I saved those files.

File: locales/en/dashboard.ftl
Comment (Vinnl) [reply locales/en/dashboard.ftl]: Ah whoops, that's only in the Premium version. d3b68cc01

File: locales-pending/dashboard-premium.ftl
Comment (rhelmer) [reply locales-pending/dashboard-premium.ftl:6]: > @rhelmer Not sure I understand.
> 
> > we decided to just land it which should make this easier.
> 
> "it" is which PR?

Sorry, I mean the re-branding PR https://github.com/mozilla/blurts-server/pull/3934 that introduces `brand-monitor-plus`.

File: src/app/components/client/stories/ProgressCard.stories.ts
Comment (Vinnl) [reply src/app/components/client/stories/ProgressCard.stories.ts:35]: Oh good point, it should not! 8c9b627dc

File: locales/en/dashboard.ftl
Comment (Vinnl) [reply locales/en/dashboard.ftl]: OK, split up for the non-US version in 1c4398234.

File: locales-pending/dashboard-premium.ftl
Comment (pdehaan) [reply locales-pending/dashboard-premium.ftl:6]: #3934 just landed and this PR might need a rebasin' since it looks like we have a few file conflicts now.

Comment (Vinnl) [reply locales-pending/dashboard-premium.ftl:6]: Yep, I'll do that after approval so people don't lose track of their existing reviews.

File: locales-pending/dashboard-premium.ftl
Comment (flodolo) [inline locales-pending/dashboard-premium.ftl:31]:
```suggestion
  working on fixing. Removals typically take 7–14 days but the most
```

Reading a book on English style, I recently realized you should use _en dashes_ for intervals
https://www.thepunctuationguide.com/en-dash.html

Turns out it's also in our style guide: https://acorn.firefox.com/latest/content/punctuation-7J3JVM4o

File: locales-pending/dashboard-premium.ftl
Comment (flodolo) [inline locales-pending/dashboard-premium.ftl:40]:
```suggestion
modal-exposure-type-description = { $data_broker_sites_total_num ->
   *[other] We search for you in all known data breaches and { $data_broker_sites_total_num } data broker sites that sell your personal info. Here are the two types of exposures we find:
}
```

Not really important as this remains en-US only, but worth fixing.

File: locales-pending/dashboard-premium.ftl
Comment (flodolo) [inline locales-pending/dashboard-premium.ftl:43]: "publicly publishing" sounds repetitive?

File: locales-pending/dashboard-premium.ftl
Comment (flodolo) [inline locales-pending/dashboard-premium.ftl:46]:
```suggestion
  In both cases, removals typically take 7–14 days. Some can take longer, while others can happen within the hour.
```

File: locales-pending/dashboard-premium.ftl
Comment (flodolo) [inline locales-pending/dashboard-premium.ftl:52]:
```suggestion
modal-exposure-status-description-premium = { $data_broker_sites_total_num ->
   *[other] We search for exposures in all known data breaches and { $data_broker_sites_total_num } data broker sites that sell your personal info. Your exposures will have one of the following statuses:
}
```

Also removing the unnecessary line break.

File: locales-pending/dashboard-premium.ftl
Comment (flodolo) [inline locales-pending/dashboard-premium.ftl:65]: Remind me: are we going to keep this (and the string above) as-is for the time being?

File: locales-pending/dashboard-premium.ftl
Comment (flodolo) [inline locales-pending/dashboard-premium.ftl:81]: We don't seem very consistent in how to spell "in progress" vs "in-progress"?

File: locales/en/app.ftl
Comment (flodolo) [inline locales/en/app.ftl:931]:
```suggestion
user-menu-settings-tooltip = Configure { -brand-mozilla-monitor }
```

Since we have the rebranding in place, when rebasing we should use the new term.

File: locales/en/app.ftl
Comment (flodolo) [inline locales/en/app.ftl:933]:
```suggestion
user-menu-help-tooltip = Get help using { -brand-mozilla-monitor }
```

File: locales/en/app.ftl
Comment (flodolo) [inline locales/en/app.ftl:935]:
```suggestion
user-menu-signout-tooltip = Sign out of { -brand-mozilla-monitor }
```

File: locales/en/exposure-card.ftl
Comment (flodolo) [inline locales/en/exposure-card.ftl:9]:
```suggestion
# Domain refers to the company's website
exposure-card-company = Company domain
```

I assume this refers to the website's domain name?

File: locales/en/fix.ftl
Comment (flodolo) [inline locales/en/fix.ftl:150]:
```suggestion
security-recommendation-email-step-one = Don’t click on links in emails from unknown senders; if it appears to be from trusted source, call directly to confirm
```

**Review by flodolo — APPROVED** [review]

File: locales-pending/dashboard-premium.ftl
Comment (Vinnl) [reply locales-pending/dashboard-premium.ftl]: 12ea98cb0

File: locales-pending/dashboard-premium.ftl
Comment (Vinnl) [reply locales-pending/dashboard-premium.ftl]: 86fd34363

File: locales-pending/dashboard-premium.ftl
Comment (Vinnl) [reply locales-pending/dashboard-premium.ftl:46]: I think that that's for emphasis that your data is out there :)

File: locales-pending/dashboard-premium.ftl
Comment (Vinnl) [reply locales-pending/dashboard-premium.ftl]: 12ea98cb0

File: locales-pending/dashboard-premium.ftl
Comment (Vinnl) [reply locales-pending/dashboard-premium.ftl]: 02f9fc56d

File: locales-pending/dashboard-premium.ftl
Comment (Vinnl) [reply locales-pending/dashboard-premium.ftl:70]: Yeah, I asked about it, but since it's US-only (and I think harder to break up into three parts), [it's being left as-is for now](https://mozilla.slack.com/archives/C0460QALW69/p1704304683210789?thread_ts=1704214729.664269&cid=C0460QALW69).

File: locales-pending/dashboard-premium.ftl
Comment (Vinnl) [reply locales-pending/dashboard-premium.ftl:86]: I believe (but I'm not a native speaker) that "in-progress" is an adjective (i.e. "an in-progress scan"), whereas "in progress" is a sentence fragment ("the scan is in progress").

File: locales/en/app.ftl
Comment (Vinnl) [reply locales/en/app.ftl]: d925a2e19

File: locales/en/app.ftl
Comment (Vinnl) [reply locales/en/app.ftl]: d925a2e19

File: locales/en/app.ftl
Comment (Vinnl) [reply locales/en/app.ftl]: d925a2e19

File: locales/en/exposure-card.ftl
Comment (Vinnl) [reply locales/en/exposure-card.ftl]: I tried to check how it was used, but it looks like this (and `exposure-card-company-logo`) weren't used any more, so I removed them: b3d0d6430

File: locales/en/fix.ftl
Comment (Vinnl) [reply locales/en/fix.ftl]: c5283e8f0

File: locales-pending/dashboard-premium.ftl
Comment (flodolo) [inline locales-pending/dashboard-premium.ftl:107]:
```suggestion
false-door-test-phase-2-content-part-two-dashboard = Automatically remove data from broker sites.
```

Just noticed this one has a weird non-breaking space at the end.

File: locales-pending/dashboard-premium.ftl
Comment (flodolo) [reply locales-pending/dashboard-premium.ftl:86]: Right, but we're only using in-progress in two places (the category itself is "In progress", sometimes title case).

The other usage is even weirder

> This chart does not include any exposures that are in-progress of being auto-removed.

File: locales-pending/dashboard-premium.ftl
Comment (Vinnl) [reply locales-pending/dashboard-premium.ftl]: Oh huh, how did that get there? 2677cf78f

File: locales-pending/dashboard-premium.ftl
Comment (Vinnl) [reply locales-pending/dashboard-premium.ftl:86]: Asked about that [here](https://mozilla.slack.com/archives/CEV7MGSQH/p1704359607505119).

Do we want to wait for an answer, or can we fix it up later? (Since I don't expect it to influence translations, right?)

Comment (flodolo) [reply locales-pending/dashboard-premium.ftl:86]: Anything in `locales-pending` can be fixed later.

Comment (Vinnl) [reply locales-pending/dashboard-premium.ftl:86]: Ah heh right, I'll get to rebasing then :)

File: locales-pending/dashboard-premium.ftl
Comment (Vinnl) [reply locales-pending/dashboard-premium.ftl]: Alright, all conflicts are resolved, tests are fixed, approvals are in, so with a final rebase I think this is ready for merging in!

**Pull request merged**
